### PR TITLE
[Docs]- add ui doc for doris connector

### PR DESCRIPTION
**Pull request by chyueyi**
### Describe your changes:

Doc #14144 

#
### Type of change:

- [ ] Bug fix
- [ ] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/public/locales/en-US/Database/Doris.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+# Doris
+
+In this section, we provide guides and references to use the Doris connector.
+
+## Requirements
+
+You can find further information on the Doris connector in the [docs](https://docs.open-metadata.org/connectors/database/doris).
```
Comment (ulixius9) [inline openmetadata-ui/src/main/resources/ui/public/locales/en-US/Database/Doris.md:7]: Thanks for the pr @chyueyi, this link would currently not work as we have not added the docs in the docs dir, do you plan add the same with this pr? 🙇

**Review by ulixius9 — APPROVED** [review]

File: openmetadata-ui/src/main/resources/ui/public/locales/en-US/Database/Doris.md (hunk 1 above)
Comment (chyueyi) [reply openmetadata-ui/src/main/resources/ui/public/locales/en-US/Database/Doris.md:7]: OK @ulixius9

**Comment by chyueyi** [comment]
Please review this pr @ulixius9

**Review by ulixius9 — APPROVED** [review]
Thanks @chyueyi

**Comment by ShilpaVernekar** [comment]
Hi @chyueyi 
Hope you are doing well.
Please can we connect on [OpenMetadata Slack](https://openmetadata.slack.com/) to discuss further about your code contribution for Doris connector. Thanks.

**Pull request merged**